src/Sarif.Converters/PREFastConverter.cs

```diff
@@ -99,6 +106,17 @@ private Result CreateResult(Defect defect)
             return result;
         }
 
+        private void TryAddLogicalLocation(string name, string decoratedName)
```

Comment:
This isn't a `Try` function -- those return `false` if they can't do what you ask. This is just an overload of `AddLogicalLocation`.

Reply:
Fixed